rb/lib/selenium/webdriver/common/service_manager.rb

```diff
@@ -110,6 +110,8 @@ def start_process
       end
 
       def stop_process
+        return if process_exited?
```

Comment:
to avoid an error here why don't we instead check that there is no process running?

i.e. `return unless process_running?` Because that checks the iVar presence as well as the nil check, and an alive check.

EDIT: Although in hindsight they're both kinda similar.

Reply:
When a process is exited, the `stop_process` method is called via `ensure` on [L69](https://github.com/SeleniumHQ/selenium/blob/trunk/rb/lib/selenium/webdriver/common/service_manager.rb#L69), triggered by `return if process_exited?` on [L63](https://github.com/SeleniumHQ/selenium/blob/trunk/rb/lib/selenium/webdriver/common/service_manager.rb#L63), so I wrote `return if process_exited?`.